apps/snmp_exporter/app.yaml

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "/snmp_exporter",
+    "cmd": null,
+    "cpus": 1,
+    "mem": 1024,
+    "disk": 0,
+    "gpus": 0,
+    "instances": 1,
+    "executor": "",
+    "requirePorts": false,
+    "constraints": [
+        [
+            "secrets",
+            "LIKE",
+            "true"
+        ],
+    ],
+    "acceptedResourceRoles": ['*'],
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "docker.ocf.berkeley.edu/snmp_exporter",
+            "privileged": false,
+            "parameters": [],
+            "forcePullImage": false
+        },
+        "portMappings": [
+            {
+                "containerPort": 9116,
+                "hostPort": 0,
+                "servicePort": 10013,
```

Comment:
this port is already in use by sourcegraph, I think

Reply:
Yeah, I commented on https://github.com/ocf/puppet/commit/64e1edc6ba8089cda23e2638a8aab5cde8888154 about this, I think it should use 10014 or maybe some port a bit higher to avoid conflicting with sourcegraph language intelligence services.